006_team_4_agent_design/006_team_4_agent_design.tex

```diff
@@ -1 +1,311 @@
+
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=0.85\textwidth]{006_team_4_agent_design/assets/JeremyPittTrustFramework.jpg}
+    \caption{An illustration showing the formal model of the Trust Framework. \cite{JeremyPittSlides}}
+    \label{fig:TrustFramework}
+\end{figure}
+
+Trust score acted as an element of symbiosis between the machine learning-based evolutionary algorithm that the agent was originally employed with and a self-organising strategy. Additionally, it allowed the agent to improve on it's relatively static structure to become more dynamic so as to adapt to the changes in it's surrounding environment. This was advantageous as the agent was able to use its pre-defined metrics in tandem with trust score and cravings to adapt within the tower depending on its social construction of reality. The trust score enacted the formal model (as shown in \Cref{fig:TrustFramework}) of a trust framework whereby the opportunity to trust is employed through communication and action, whilst the quantity decision is made through every individual agent strategy. Through local trust based modelling of a global trust score, the agent is able to implement the outcome utilities portion of the trust framework.
+
+\section{Experimentation and Results}
+
+\subsection{Baseline Comparisons}
+In order to evaluate the performance of our agent, it was compared to two baselines - the default agent and the random agent. The time taken for the system to stabilise (i.e time taken for the agents to organise into a structure that led to no future deaths) into a viable social order was measured. Tests with homogeneous populations of Team 4 agents, default agents and random agents were conducted, which highlighted that the two baselines never stabilised, whilst the Team 4 agent did. This was evident from \Cref{fig:baselinecomps} which showed the number of deaths for the Team 4 agents eventually stagnating however, this did not occur for the other two agent types. This was because the agent was able to build a trustworthy local perception which resulted in a positive outlook of the environment, allowing all agents to tend towards selflessness. This selfless behaviour enacted utilitarianism as the agents were able to maximise the average food left on the platform after a day had passed, ensuring that there was no wastefulness, whilst the overall population was still able to survive. 
```

Comment:
i.e -> i.e.
```suggestion
In order to evaluate the performance of our agent, it was compared to two baselines - the default agent and the random agent. The time taken for the system to stabilise (i.e. time taken for the agents to organise into a structure that led to no future deaths) into a viable social order was measured. Tests with homogeneous populations of Team 4 agents, default agents and random agents were conducted, which highlighted that the two baselines never stabilised, whilst the Team 4 agent did. This was evident from \Cref{fig:baselinecomps} which showed the number of deaths for the Team 4 agents eventually stagnating however, this did not occur for the other two agent types. This was because the agent was able to build a trustworthy local perception which resulted in a positive outlook of the environment, allowing all agents to tend towards selflessness. This selfless behaviour enacted utilitarianism as the agents were able to maximise the average food left on the platform after a day had passed, ensuring that there was no wastefulness, whilst the overall population was still able to survive. 
```